### PR TITLE
fix for #21

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Comment by stefhak** [comment]
LGTM.

**Comment by stefhak** [comment]
@martinthomson can you review this PR? (Should be quickly done)

**Comment by martinthomson** [comment]
LGTM

File: webrtc.html [hunk 1]
```diff
@@ -2095,14 +2095,14 @@
...
-            <p>An RTCSdpType of "offer" indicates that a description should be
+            <p>An RTCSdpType of "offer" indicates that a description MUST be
             treated as an [[!SDP]] offer.</p>
           </dd>
 
           <dt>pranswer</dt>
 
           <dd>
-            <p>An RTCSdpType of "pranswer" indicates that a description should
+            <p>An RTCSdpType of "pranswer" indicates that a description MUST
```
Comment (alvestrand) [inline webrtc.html]: Not happy with "MUST" here; what about "is intended to be treated as"?

Comment (stpeter) [inline webrtc.html]: That does seem better in this instance.

**Pull request merged**